app/src/components/Chats/ChatPreview.tsx

```diff
@@ -61,77 +66,102 @@ export default function ChatPreview({
   const firstMessage = sortedMessages ? sortedMessages[0] : null
   const lastMessage = sortedMessages && sortedMessages.length > 1 ? sortedMessages.at(-1) : null
   const repliesCount = sortedMessages ? sortedMessages.length - 2 : 0
+  const showName =
+    // const messageUser = lastMessage ? chatUsers[lastMessage.authorId] : null
 
-  // const messageUser = lastMessage ? chatUsers[lastMessage.authorId] : null
+    useEffect(() => {
+      setUnreadMessages(getUnreadMessagesCount(chat.id))
+      // console.log(chatUsers[firstMessage.authorId].name)
+    }, [unreadChats])
 
-  useEffect(() => {
-    setUnreadMessages(getUnreadMessagesCount(chat.id))
-  }, [unreadChats])
+  const disableChat = unreadMessages === 0
 
   return (
-    <div
-      className='w-full h-52 flex items-center pl-5 border-b-2 border-border-color hover:cursor-pointer hover:bg-gray-100'
-      onClick={onClick}
-    >
-      <div className='w-full'>
-        <div className='relative flex items-center h-10 gap-2 mb-3 '>
-          {unreadMessages ? (
-            <div className='h-7 w-7 bg-red-400 border-2 border-black flex items-center justify-center'>
-              <span className='font-semibold'>+{unreadMessages}</span>
-            </div>
-          ) : null}
-          {chat.content ? (
-            <div className='h-full flex gap-2 items-center font-monaspace whitespace-nowrap'>
-              <div className='h-full w-[3px] bg-gray-500' />
-              <div>{editorContent || chat.content}</div>
-            </div>
-          ) : null}
-          <Menu shadow='md' width={200}>
-            <Menu.Target>
-              <i
-                onClick={(e) => e.stopPropagation()}
-                className='z-40 ri-more-2-fill text-2xl absolute top-1 right-5'
+    // <div
+    //   className='w-full flex border-border-color hover:cursor-pointer hover:bg-gray-100 pr-7  py-3'
+    //   onClick={onClick}
+    // >
+    <div className='flex w-full'>
+      <div
+        className={`w-2 h-full border-r-2 border-border-color ${
```

Comment:
let's use `clsx` for it
should be like that:
`className={clsx('w-2 h-full border-r-2 border-border-color', isChatDisabled ? 'bg-[#EFEFEF]' : 'bg-[#DBF7CA]')}`